twoliter/embedded/Makefile.toml

```diff
@@ -329,11 +329,10 @@ dependencies = ["setup"]
 script_runner = "bash"
 script = [
 '''
-if [ -f "${BUILDSYS_SOURCES_DIR}/Cargo.toml" ]; then
-  cargo fetch --locked --manifest-path "${BUILDSYS_SOURCES_DIR}/Cargo.toml"
-fi
-
-cargo fetch --locked --manifest-path "${BUILDSYS_ROOT_DIR}/Cargo.toml"
+for ws in sources variants .; do
+  [ -s "${BUILDSYS_ROOT_DIR}/${ws}/Cargo.toml" ] || continue
```

Comment:
Why is this `-s` and not `-f`?

Reply:
I usually prefer `-s` over `-f` for cases where a non-empty file is expected. `cargo` doesn't like empty manifests.